src/index.js

```diff
@@ -0,0 +1,235 @@
+    .then((response) => response.json())
+    .then((json) => {
+      const meal = json.meals[0];
+      const popupHtml = `
+            <div class="container">
+            <i class="fas fa-times fa-2x" id="go-back"></i>
+                <div id="img-comment" class="img-comment">
+                <img src="${meal.strMealThumb}" alt="meal-img">
+                <h5>${meal.strMeal}</h5>
+                </div>
+                <div id="info-item-comment" class="item-info">
+                <a href=${meal.strSource} target="_blank">See more about this meal <i class="fas fa-arrow-right"></i></a>
+                </div>
+                <h4 id="comments-header">Comments By previous Visitors</h4>
+                <div id="comments" class="comments"></div>
+                <div id="form-comment" class="form-comment">
+                <form id="comment-form">
+                <div class="form-group mb-3">
+                <input type="text" class="form-control" id="your-name" name="name" placeholder="Your name" required>
+                </div>
+                <div class="form-group mb-3">
+                <textarea class="form-control" id="your-comments" name="comment" placeholder="Your comment" required></textarea>
+                </div>
+                <button type="submit" class="btn btn-outline-secondary submit-btn">Comment</button>
+                </form>
+
+                </div>
+                </div>
+                `;
+      popupNode.innerHTML = popupHtml;
+      loadComments(meal.idMeal);
+
+      const commentForm = document.querySelector('#comment-form');
+      commentForm.addEventListener('submit', (e) => {
+        e.preventDefault();
+        const commentObj = {
+          item_id: meal.idMeal,
+          username: commentForm.name.value,
+          comment: commentForm.comment.value,
+        };
+        uploadComment(commentObj);
+        commentForm.name.value = '';
+        commentForm.comment.value = '';
+        setTimeout(() => { loadComments(meal.idMeal); }, 1000);
+      });
+
+      const header = document.querySelector('header');
+      const main = document.querySelector('.row');
+      const footer = document.querySelector('footer');
+      const popupComment = document.querySelector('#popup-comment');
+
+      const goBack = document.querySelector('#go-back');
+      goBack.addEventListener('click', () => {
+        popupNode.innerHTML = '';
+        header.style.filter = 'none';
+        main.style.filter = 'none';
+        footer.style.filter = 'none';
+        popupComment.style.display = 'none';
+      });
+    });
+};
+
+const getFood = async () => {
+  const response = await fetch('https://www.themealdb.com/api/json/v1/1/filter.php?a=Chinese');
+  response.json().then((json) => {
+    const itemArr = json.meals;
+    console.log(json);
+    const dishNum = document.querySelector('#dish-num');
+    dishNum.textContent = `Dishes (${itemArr.length})`;
```

Comment:
- [ ] You are expected, to still create a specific function to calculate counter numbers on each page even if the API provides that information. Let's write a function to handle that

- [ ]  Note that the counters functions (dishes counter and comment counter) should get covered with unit tests as well. Currently, your tests don't cover them